exchange/bidder.go

```diff
@@ -87,20 +88,27 @@ type pbsOrtbSeatBid struct {
 //
 // The name refers to the "Adapter" architecture pattern, and should not be confused with a Prebid "Adapter"
 // (which is being phased out and replaced by Bidder for OpenRTB auctions)
-func adaptBidder(bidder adapters.Bidder, client *http.Client, cfg *config.Configuration, me pbsmetrics.MetricsEngine) adaptedBidder {
+func adaptBidder(bidder adapters.Bidder, client *http.Client, cfg *config.Configuration, me pbsmetrics.MetricsEngine, name openrtb_ext.BidderName) adaptedBidder {
 	return &bidderAdapter{
 		Bidder:      bidder,
+		BidderName:  name,
 		Client:      client,
 		DebugConfig: cfg.Debug,
-		me:          me,
+		me:          bidderMetrics{engine: me, disabled: cfg.Metrics.Disabled},
 	}
 }
 
 type bidderAdapter struct {
 	Bidder      adapters.Bidder
+	BidderName  openrtb_ext.BidderName
 	Client      *http.Client
 	DebugConfig config.Debug
-	me          pbsmetrics.MetricsEngine
+	me          bidderMetrics
+}
+
+type bidderMetrics struct {
+	engine   pbsmetrics.MetricsEngine
+	disabled config.DisabledMetrics
```

Comment:
I would normally expect a property called `disabled` to be a bool. `disabledMetrics` perhaps?

Reply:
Makes total sense. Corrected.